benchmark/scenarios/test/test_rate_measurer.py

```diff
@@ -0,0 +1,147 @@
+from flocker.testtools import TestCase
+
+from .._rate_measurer import RateMeasurer
+
+
+class RateMeasurerTest(TestCase):
+    """
+    RateMeasurer tests.
+    """
+
+    def send_requests(self, rate_measurer, num_requests, num_samples):
+        """
+        Helper function that will send the desired number of request.
+
+        :param rate_measurer: The `RateMeasurer` we are testing.
+        :param num_requests: The number of request we want to send.
+        :param num_samples: The number of samples to collect.
+        """
+        for i in range(num_samples * num_requests):
+            rate_measurer.request_sent()
+
+    def receive_requests(self, rate_measurer, num_requests, num_samples):
+        """
+        Helper function that will receive the desired number of requests.
+
+        :param rate_measurer: The `RateMeasurer` we are testing.
+        :param num_requests: The number of request we want to receive.
+        :param num_samples: The number of samples to collect.
+        """
+        ignored = ""
```

Comment:
Bytes or unicode?  Please make this explicit.  Either use `u""`/`b""` or use the `unicode_literals` future import in this module (and fix any other strings).